libs/portal/features/profile/ui/questions.tsx

```
@@ -0,0 +1,427 @@
+
+const initialSocialMediaHandles = {
+  discord: '',
+  instagram: '',
+  gitHub: '',
+  behance: '',
+};
+
+export function Questions({ user, status }: ProfileFormProps) {
+  const [isStudent, setIsStudent] = useState<boolean>(false);
+  // eslint-disable-next-line no-unused-vars
+  const [socialMediaHandles, setSocialMediaHandles] = useState<
```

ESLint

[error] 'setSocialMediaHandles' is assigned a value but never used. Allowed unused vars must match /^_/u.

(unused-imports/no-unused-vars)